examples/rac-spectrum-tailwind/src/components/StarRatingGroup.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react";
+import {
+  Group,
+  Label,
+  Radio,
+  RadioGroup,
+  RadioGroupProps,
+} from "react-aria-components";
+
+interface StarRatingGroupProps extends Omit<RadioGroupProps, "children"> {
+  ratingCount?: number;
+  value?: string;
+  defaultValue?: string;
+  onChange?: (value: string) => void;
+  isEmphasized?: boolean;
+  label?: string;
+}
+
+export function StarRatingGroup({
```

Comment:
i wish i could use this one with arrow keys

Reply:
Fixed, thanks!